test/lib/usd/importExport/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+set(TARGET_NAME IMPORT_EXPORT_TEST)
+
+set(test_script_files
+    testUsdExportAsClip.py
+    testUsdExportCamera.py
+    testUsdExportColorSets.py
+    testUsdExportConnected.py
+    testUsdExportDisplayColor.py
+    testUsdExportEulerFilter.py
+    testUsdExportFilterTypes.py
+    testUsdExportFrameOffset.py
+    testUsdExportInstances.py
+    testUsdExportLocator.py
+    testUsdExportMesh.py
+    testUsdExportNurbsCurve.py
+    testUsdExportOpenLayer.py
+    testUsdExportOverImport.py
```

Comment:
Hey @ppt-adsk , 

I did a clean build of maya-usd with your changes on Windows.  Ran all the tests and ctest reported that all tests pass:

```
100% tests passed, 0 tests failed out of 104

Label Time Summary:
importExport         = 667.42 sec*proc (39 tests)
ufe                  = 390.75 sec*proc (19 tests)
usdPreviewSurface    =  33.33 sec*proc (2 tests)

Total Test time (real) =  85.62 sec
```

I then ran the tests again but ctest reports that `testUsdExportOverImport ` failed. Do you see this behavior on your side as well?

Here is what -VV shows:
```
40: ======================================================================
40: FAIL: testImportModifyAndExportCubeModel (testUsdExportOverImport.testUsdExportOverImport)
40: ----------------------------------------------------------------------
40: Traceback (most recent call last):
40:   File "testUsdExportOverImport.py", line 73, in testImportModifyAndExportCubeModel
40:   File "testUsdExportOverImport.py", line 50, in _ValidateUsdBeforeExport
40: AssertionError: invalid null prim is not true
40:
40: ----------------------------------------------------------------------
40: Ran 1 test in 0.323s
40:
40: FAILED (failures=1)
1/1 Test #40: testUsdExportOverImport ..........***Failed    6.35 sec
```